Extension/walkthrough/install-clang-macos.md

```diff
@@ -0,0 +1,13 @@
+# Install a C++ compiler on macOS
+
+If you're doing C++ development for macOS, we recommend installing the Clang compiler. All you need to do is run the following command in a Terminal window to install the command line developer tools:
+
+```bash
+xcode-select --install
+```
+
+Then, to verify that clang is installed, run the following command in a Terminal window. You should see a copyright message with the version and basic usage description.
```

Comment:
I don't see a copyright message or basic usage:
```
~$ clang --version
clang version 10.0.0-4ubuntu1
Target: x86_64-pc-linux-gnu
Thread model: posix
InstalledDir: /usr/bin
```
just the version info.

Reply:
Thanks for checking. Changing it to "You should see a message with information about the version of Clang you're using."